src/map/message.cpp

```diff
@@ -94,10 +94,11 @@ namespace message
         }
         case MSG_CHAT_TELL:
         {
+            CCharEntity* PSender = zoneutils::GetChar(RBUFL(extra->data(), 0));
```

Comment:
you'll wanna look up their gm level in db in case they dont exist in the zone

Reply:
Reworking this to send GM level in packet, per-discord discussion.